bgcompute.py

```diff
@@ -30,6 +37,24 @@ def bgcompute():
         )
 
 
+def bgcompute_update_election_jurisdictions_file() -> int:
+    elections = (
+        Election.query.join(File, File.id == Election.jurisdictions_file_id)
+        .filter(
+            Election.jurisdictions_file_id != None, File.processing_started_at == None
+        )
+        .options(joinedload(Election.jurisdictions_file))
```

Comment:
nit: simpler code might be to query the `File` table joining against `Election` and adding the `Election` entity.

Reply:
Good call. That is better.